botx/client/notifications_api/direct_notification.py

```diff
@@ -80,6 +81,11 @@ def from_domain(
             assert not file.is_async_file, "async_files not supported"
             api_file = BotXAPIAttachment.from_file_attachment(file)
 
+        if len(body) > MAXIMUM_TEXT_LENGTH:
+            raise ValueError(
+                f"Message body length exceeds {MAXIMUM_TEXT_LENGTH} symbols.",
```

Comment:
Maybe rename to `MAX_NOTIFICATION_BODY_LENGTH`?

Reply:
I agree. Fix.